test/Sema/accessibility_shared_private.swift

```diff
@@ -0,0 +1,76 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: %utils/split_file.py -o %t %s
+// RUN: %target-swift-frontend -swift-version 4 -typecheck %t/declarations.swift %t/other_file_extensions.swift -verify
```

Comment:
`VerifyDiagnosticConsumer` defines the `expected-[]` behavior. It would be great if that syntax supported the idea of a prefix, then this would be pretty trivial. Any other ideas on how to update this test @jrose-apple ?